### PR TITLE
Add note about MSRV in README

### PR DESCRIPTION
**Pull request by Gordon-F**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 The shader translation library for the needs of [wgpu](https://github.com/gfx-rs/wgpu) and [gfx-rs](https://github.com/gfx-rs/gfx) projects.
 
+**Minimum Supported Rust Version:** `1.43.0`
```
Comment (kvark) [inline README.md:11]: do you think we could have a real github badge for this instead of a note? I think I saw some in other projects

Comment (Gordon-F) [reply README.md:11]: I saw only generated with `shields.io`. Like this one:

Comment (kvark) [reply README.md:11]: looks neat!

Comment (Gordon-F) [reply README.md]: Done. Rendered - https://github.com/Gordon-F/naga/blob/patch-2/README.md.

**Review by kvark — APPROVED** [review]

**Pull request merged**